package/yast2.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Wed Mar 25 12:12:00 UTC 2020 - Knut Anderssen <kanderssen@suse.com>
+
+- Force a cache clean and read of the current network interfaces
```

Comment:
I would write it as "Reread interfaces configuration after writing it" which seems simple and still not user oriented. So maybe adding something like "to avoid wrong values when reopen dialog in installation" ?